_posts/en/meetings/2018-07-19-meeting.md

```diff
@@ -0,0 +1,304 @@
+after Scaling Bitcoin, October 8-10, and to organize it in a similar
+fashion as the last one in NYC."
+
+Several people thanked Lee.
+
+**Conclusion:** Lee said, "I plan to send out a survey to collect some
+feedback.  If anyone has specific ideas or suggestions, please feel free
+to contact me."
+
+## Humor
+
+{% highlight text %}
+          <sipa> what about #13666 ?
+<provoostenator> What's in a number?
+          <sipa> 13 and 666, can't beat those odds
+        <wumpus> niice
+      <achow101> it was completely planned, obviously
+          <sipa> in some timezones it was also opened on
+                 friday the 13th
+                 oh, no
+ <jonasschnelli> I hope no black cat was sitting on the
+                 keyboard during coding
+{% endhighlight %}
+
+
+## Participants
+
+| IRC nick        | Name/Nym                  |
+|-----------------|---------------------------|
+| wumpus          | [Wladimir van der Laan][] |
+| sipa            | [Pieter Wuille][]         |
+| jonasschnelli   | [Jonas Schnelli][]        |
+| achow101        | [Andrew Chow][]           |
+| luke-jr         | [Luke Dashjr][]           |
+| gmaxwell        | [Gregory Maxwell][]       |
+| moneyball       | [Steve Lee][]             |
+| provoostenator  | [Sjors Provoost][]        |
+| jnewbery        | [John Newbery][]          |
+| cfields         | [Cory Fields][]           |
+| lclc            | [Lucas Betschart][]       |
+| ken2812221      | [Chun Kuan Lee][]         |
+| kanzure         | [Bryan Bishop][]          |
+
+## Disclaimer
+
+This summary was compiled without input from any of the participants in
+the discussion, so any errors are the fault of the summary author and
+not the discussion participants.  In particular, quotes taken from the
+discussion had their capitalization, punctuation, and spelling modified
+to produce consistent sentences.  Bracketed words and fragments, as well
+as background narratives and exposition, were added by the author of
+this summary and may have accidentally changed the meaning of some
+sentences.  If you believe any quote was taken out of context, please
+[open an issue](https://github.com/bitcoin-core/bitcoincore.org/issues/new)
+and we will correct the mistake.
+
+[bbm log]: https://botbot.me/freenode/bitcoin-core-dev/msg/102255898/
+[mb minutes]: http://www.erisian.com.au/meetbot/bitcoin-core-dev/2018/bitcoin-core-dev.2018-07-19-19.02.html
+
+[current high-priority PRs]: https://github.com/bitcoin/bitcoin/projects/8
+
+{% assign log = "http://www.erisian.com.au/meetbot/bitcoin-core-dev/2018/bitcoin-core-dev.2018-07-19-19.02.log.html" %}
+[mb log]: {{log}}
+[log hipri]: {{log}}#l-13
+[log rpc coin selection]: {{log}}#l-77
+[log scantxoutset api]: {{log}}#l-173
+[log seeder]: {{log}}#l-207
+[log bco]: {{log}}#l-231
+[log coredev meet]: {{log}}#l-279
+
+[meet osd]: /en/meetings/2018/06/28/#output-script-descriptors
+[osd gist]: https://gist.github.com/sipa/e3d23d498c430bb601c5bca83523fa82
```

Comment:
I sometimes worry that gists aren't stable links, since they feel more ephemeral. There's not really any alternative though, and better to link to this now and risk having it break in future than not link.